docs/Inleiding.md

```diff
@@ -0,0 +1,43 @@
+# Inleiding
+
+## Historie
+
+VNGRealisatie is enkele jaren geleden begonnen met het inzetten van Imvertor als tool om de kwaliteit en de consistentie van de informatiemodellen te controleren. Tijdens dit traject rol ontstond het idee om op basis het informatiemodel en een omzettings-algoritme de basisentiteiten voor StUF te genereren. Toen dit realiseerbaar bleek te zijn bleek dat daar veel consistentie-voordelen aan zaten. Om die voordelen door te trekken naar de daadwerkelijke koppelvlakken is vervolgens ook een methodiek ontwikkeld om de koppelvlakken zelf te modelleren en genereren.
+Dit traject viel samen met de verschuiving van de focus binnen VNG Realisatie van StUF-XML naar Rest/Json-API's.
+Om deze aanpak toepasbaar te maken voor zowel Stuf-XML als Rest/Json API's is er een drielagen-model opgezet. In de afgelopen 2 jaar is de focus volledig verlegd naar het genereren van Rest/Json API-specificaties.
```

Comment:
Wijzig `... voor zowel Stuf-XML als ...`
in `... voor zowel StUF-XML als ...`

Reply:
Aangepast